### DISCUSSION TITLE
Multiple WINEDLLOVERRIDES variables

### DISCUSSION THREAD

**Discussion by sendnonsensehere** [Heroic-Games-Launcher/HeroicGamesLauncher#3980] [Q&A]
Hi, I've set my game to run two variables for WINEDLLOVERRIDES: d3d8=n & msvcrt=n,b, but no matter how I tried to phrase the two together, it only ever accepts the one I put first. Any good ideas on how I can word the two to run together?

**Comment by xinyifly** [comment]
`WINEDLLOVERRIDES="d3d8=n;msvcrt=n,b"`
